lib/recurly/resource.rb

```diff
@@ -144,17 +144,29 @@ class Invalid < API::UnprocessableEntity
       # @return [Resource, nil] The invalid record.
       attr_reader :record
 
-      def initialize(record_or_message)
-        set_message case record_or_message
-        when Resource
-          @record = record_or_message
-          record_or_message.errors.map { |k, v| "#{k} #{v * ', '}" }.join '; '
+      def initialize(message)
+        if message.is_a? Resource
+          @record = message
+          set_message(record_to_message)
         else
-          record_or_message
+          set_message(message)
         end
       end
+
+      private
+
+      def record_to_message
+        @record.errors.map do |k, v|
+          if k == "base"
+            "#{v.join(', ')}"
```

Comment:
Yeah, I looked over this

https://github.com/recurly/recurly-client-ruby/pull/268